compiler warning on drakeShapes

just saw this one.  @avalenzu - can you take a look?

```
[ 59%] Building CXX object systems/plants/shapes/CMakeFiles/drakeShapes.dir/Geometry.cpp.o
/tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h: In member function 'virtual void DrakeShapes::MeshPoints::getBoundingBoxPoints(Eigen::Matrix3Xd&) const':
/tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h:71:5: warning: '*((void*)& min_pos +24)' may be used uninitialized in this function [-Wuninitialized]
/tmp/drake-master/drake-19757/systems/plants/shapes/Geometry.cpp:285:14: note: '*((void*)& min_pos +24)' was declared here
/tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h:71:5: warning: '*((void*)& max_pos +24)' may be used uninitialized in this function [-Wuninitialized]
/tmp/drake-master/drake-19757/systems/plants/shapes/Geometry.cpp:286:14: note: '*((void*)& max_pos +24)' was declared here
/tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h: In member function 'virtual void DrakeShapes::Mesh::getBoundingBoxPoints(Eigen::Matrix3Xd&) const':
/tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h:71:5: warning: '*((void*)& min_pos +24)' may be used uninitialized in this function [-Wuninitialized]
/tmp/drake-master/drake-19757/systems/plants/shapes/Geometry.cpp:260:16: note: '*((void*)& min_pos +24)' was declared here
/tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h:71:5: warning: '*((void*)& max_pos +24)' may be used uninitialized in this function [-Wuninitialized]
/tmp/drake-master/drake-19757/systems/plants/shapes/Geometry.cpp:261:16: note: '*((void*)& max_pos +24)' was declared here
```

Will do.

On Tue, Jul 14, 2015 at 7:33 AM Russ Tedrake <notifications@github.com>
wrote:

> just saw this one. @avalenzu <https://github.com/avalenzu> - can you take
> a look?
>
> [ 59%] Building CXX object systems/plants/shapes/CMakeFiles/drakeShapes.dir/Geometry.cpp.o
> /tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h: In member function 'virtual void DrakeShapes::MeshPoints::getBoundingBoxPoints(Eigen::Matrix3Xd&) const':
> /tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h:71:5: warning: '*((void*)& min_pos +24)' may be used uninitialized in this function [-Wuninitialized]
> /tmp/drake-master/drake-19757/systems/plants/shapes/Geometry.cpp:285:14: note: '*((void*)& min_pos +24)' was declared here
> /tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h:71:5: warning: '*((void*)& max_pos +24)' may be used uninitialized in this function [-Wuninitialized]
> /tmp/drake-master/drake-19757/systems/plants/shapes/Geometry.cpp:286:14: note: '*((void*)& max_pos +24)' was declared here
> /tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h: In member function 'virtual void DrakeShapes::Mesh::getBoundingBoxPoints(Eigen::Matrix3Xd&) const':
> /tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h:71:5: warning: '*((void*)& min_pos +24)' may be used uninitialized in this function [-Wuninitialized]
> /tmp/drake-master/drake-19757/systems/plants/shapes/Geometry.cpp:260:16: note: '*((void*)& min_pos +24)' was declared here
> /tmp/drake-master/build/include/eigen3/Eigen/src/Core/CommaInitializer.h:71:5: warning: '*((void*)& max_pos +24)' may be used uninitialized in this function [-Wuninitialized]
> /tmp/drake-master/drake-19757/systems/plants/shapes/Geometry.cpp:261:16: note: '*((void*)& max_pos +24)' was declared here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/issues/1172>.
>

--- a/systems/plants/shapes/Geometry.cpp
+++ b/systems/plants/shapes/Geometry.cpp
@@ -261,9 +261,9 @@ namespace DrakeShapes
       Vector3d max_pos = mesh_vertices.rowwise().maxCoeff();
       
       bbox_points.resize(Eigen::NoChange, NUM_BBOX_POINTS);
-      bbox_points << min_pos(1), min_pos(1), min_pos(1), min_pos(1), max_pos(1), max_pos(1), max_pos(1), max_pos(1),
-                     min_pos(2), min_pos(2), max_pos(2), max_pos(2), min_pos(2), min_pos(2), max_pos(2), max_pos(2),
-                     min_pos(3), max_pos(3), min_pos(3), max_pos(3), min_pos(3), max_pos(3), min_pos(3), max_pos(3);
+      bbox_points << min_pos(0), min_pos(0), min_pos(0), min_pos(0), max_pos(0), max_pos(0), max_pos(0), max_pos(0),
+                     min_pos(1), min_pos(1), max_pos(1), max_pos(1), min_pos(1), min_pos(1), max_pos(1), max_pos(1),
+                     min_pos(2), max_pos(2), min_pos(2), max_pos(2), min_pos(2), max_pos(2), min_pos(2), max_pos(2);
                      
   }
 
@@ -286,9 +286,9 @@ namespace DrakeShapes
     Vector3d max_pos = points.rowwise().maxCoeff();
      
     bbox_points.resize(Eigen::NoChange, NUM_BBOX_POINTS);
-    bbox_points << min_pos(1), min_pos(1), min_pos(1), min_pos(1), max_pos(1), max_pos(1), max_pos(1), max_pos(1),
-                   min_pos(2), min_pos(2), max_pos(2), max_pos(2), min_pos(2), min_pos(2), max_pos(2), max_pos(2),
-                   min_pos(3), max_pos(3), min_pos(3), max_pos(3), min_pos(3), max_pos(3), min_pos(3), max_pos(3);
+    bbox_points << min_pos(0), min_pos(0), min_pos(0), min_pos(0), max_pos(0), max_pos(0), max_pos(0), max_pos(0),
+                   min_pos(1), min_pos(1), max_pos(1), max_pos(1), min_pos(1), min_pos(1), max_pos(1), max_pos(1),
+                   min_pos(2), max_pos(2), min_pos(2), max_pos(2), min_pos(2), max_pos(2), min_pos(2), max_pos(2);
   }
 
 }

